account_check_printing_report_base/__manifest__.py

```diff
@@ -2,15 +2,17 @@
 # (http://www.eficent.com)
 # Copyright 2016 Serpent Consulting Services Pvt. Ltd.
 # Copyright 2017 Tecnativa.
+# Copyright 2018 iterativo.
 # License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
 
 {
     "name": "Account Check Printing Report Base",
-    "version": "11.0.1.1.1",
+    "version": "11.0.1.2.0",
     "license": "AGPL-3",
     "author": "Eficent,"
               "Serpent Consulting Services Pvt. Ltd.,"
               "Ursa Information Systems,"
+              "iterativo SRL,"
```

Comment:
@etobella @jbeficent judge if you consider this new feature enough for granting co-authorship

Reply:
@pedrobaeza is more urgent for us to have the feature available than the authorship. If that's a problem, please just help us with the merge.

Reply:
OK, then please remove it meanwhile. Don't forget any way to add you at least as contributor in the README. You deserve this.

Reply:
@mamcode please proceed as stated by Pedro

Reply:
Done!